recipes/pythonqwt/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "PythonQwt" %}
+{% set version = "0.5.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PierreRaybaut/PythonQwt/archive/v{{ version }}.tar.gz
+  sha256: ca3b4e34f51ade4681b99d73b5a9134498fee84a832e36cfd8f400b9acc07723
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - setuptools
+    - python
+    - pyqt
```

Comment:
Are these build deps?

Reply:
`setup.py` imports `qwt`, so everything needed to import it is needed at build time too.

Is `spyder` actually a dependency though? `install_requires` only lists `numpy`, and it's not in [the readme](https://github.com/PierreRaybaut/PythonQwt#requirements) either.

Reply:
According to the documentation the ```spyderlib``` is necessary for PythonQwt.